nncf/common/sparsity/statistics.py

```diff
@@ -148,19 +159,20 @@ class RBSparsityStatistics(Statistics):
     def __init__(self,
                  model_statistics: SparsifiedModelStatistics,
                  masks_consistency: float,
-                 target_level: float,
+                 target_sparsity_level: float,
                  mean_sparse_prob: float):
         """
         Initializes statistics of the RB-sparsity algorithm.
 
         :param model_statistics: Statistics of the sparsified model.
         :param masks_consistency: TODO
-        :param target_level: TODO
+        :param target_sparsity_level: Sparsity level which was applied
+            by the algorithm.
```

Comment:
Wording must be changed. A target level is a level which, in general, has not yet been completely applied by the algo; the "sparsity level which was applied by the algorithm" is a *current* level. 

Suggest: "A target level of sparsity for the algorithm to ultimately achieve". Same in other places.

Reply:
I suggest a different formulation:
"A target level of the sparsity for the algorithm for the current epoch."

Reply:
@alexsu52 @vshampor What do you think?

Reply:
Ok, it can be worded so.

Reply:
I fixed it.